src/index.js

```diff
@@ -11,15 +11,40 @@ require('./index.scss');
+            <g transform="translate(26.260870, 18.400000)">
+                <polygon id="Path-4" fill="#FFFFFF" points="11.4303847 19.6757008 5.91851402 10.5149814 10.5113344 1.8885512 16.6590031 6.53607695 13.4769737 18.3639874"></polygon>
+                <path d="M22.7315288,15.3063114 L15.0082884,1.9292865 C14.3213719,0.739588847 13.0411301,0 11.667297,0 C10.2938993,0 9.01409278,0.739153539 8.32674093,1.9288512 L0.514697696,15.4591046 C-0.171783536,16.648367 -0.171348227,18.1271093 0.514697696,19.3155011 C1.20204954,20.5047634 2.48185602,21.2447876 3.8556891,21.2447876 L19.4828227,21.2447876 L19.5159062,21.2447876 C21.6284575,21.2278106 23.3409604,19.5035543 23.3409604,17.3883911 C23.3413957,16.6453198 23.1302712,15.9279317 22.7315288,15.3063114 Z M11.6703442,4.76488494 C12.3285304,4.76488494 12.8622184,5.29813765 12.8622184,5.95632384 L12.5579379,13.0657794 C12.5274663,13.7087298 12.027297,13.9533731 11.6703442,13.9533731 C11.2702958,13.9533731 10.8180105,13.7422486 10.7827505,13.0657794 L10.4780347,5.95632384 C10.47847,5.29813765 11.012158,4.76488494 11.6703442,4.76488494 Z M12.7459911,17.5577261 C12.4630406,17.8406765 12.0721338,18.0026112 11.6703442,18.0026112 C11.2681193,18.0026112 10.8776477,17.8406765 10.5916502,17.5577261 C10.3082644,17.2747756 10.146765,16.8812569 10.146765,16.479032 C10.146765,16.0798543 10.3082644,15.6872062 10.5916502,15.4029498 C10.8776477,15.1199994 11.2711665,14.9554529 11.6703442,14.9554529 C12.0699572,14.9554529 12.4626053,15.1199994 12.7459911,15.4029498 C13.0324239,15.6867709 13.1939233,16.0798543 13.1939233,16.479032 C13.1939233,16.8812569 13.0324239,17.2743403 12.7459911,17.5577261 Z" id="Shape" fill="#D0011B" fill-rule="nonzero"></path>
+            </g>
+        </g>
+    </svg>
+    `;
+    let newVersion;
+    try {
+        const updateInfo = JSON.parse(localStorage.getItem('glimpseLatestVersion'));
+        if (updateInfo.latestVersion && updateInfo.latestVersion !== updateInfo.atTimeOfCheckVersion) {
+            newVersion = updateInfo.latestVersion;
+        }
+    } catch (e) {
+        // gulp
+    }
```

Comment:
Is it worth breaking this out into ta function. Might make it a tad more readable.